friends/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "friends",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.8.6",
```

Comment:
You're missing Axios in your dependencies list. Make sure you properly installed it following the steps in the README.

Reply:
Thanks! this is corrected.